ui/lib/components/CardTable/index.tsx

```diff
@@ -215,27 +216,23 @@ function CardTableV2(props: ICardTableV2Props) {
       noMarginTop={cardNoMarginTop}
       extra={cardExtra}
     >
-      <AnimatedSkeleton
-        showSkeleton={items.length === 0 && loading && !errorMsg}
-      >
-        {errorMsg ? (
-          <Alert message={errorMsg} type="error" showIcon />
-        ) : (
-          <div className={styles.cardTableContent}>
-            <MemoDetailsList
-              selectionMode={SelectionMode.none}
-              constrainMode={ConstrainMode.unconstrained}
-              layoutMode={DetailsListLayoutMode.justified}
-              onRenderRow={onRowClicked ? renderClickableRow : undefined}
-              columns={finalColumns}
-              items={finalItems}
-              {...restProps}
-            />
-          </div>
-        )}
+      {errMessages.map(
+        (msg, idx) =>
+          msg && <Alert key={idx} message={msg} type="error" showIcon />
+      )}
```

Comment:
I tried to move the Alert of the CardTable but found it doesn't work because the Alert displays out of the table title which is not expected.